Phoenix/Web.config

```diff
@@ -67,7 +67,7 @@
   </entityFramework>
   <connectionStrings>
     <add name="RCIDatabase"
-      connectionString="data source=adminprodsql;initial catalog=RCI;integrated security=True;MultipleActiveResultSets=True;MultiSubnetFailover=true;"
+      connectionString="data source=adminprodsql.gordon.edu;initial catalog=RCITrain;integrated security=True;MultipleActiveResultSets=True;"
```

Comment:
Do you have a plan for this to be updated automatically when pushing to production?

Reply:
There is no "push to production" in this project. Everything is done with a Visual Studio publish. Please see Web.Release.Config for the production config.